dashboard/cron.py

```diff
@@ -233,9 +223,9 @@ def update_with_bq_access(self):
                                 query = '''select CAST(SUBSTR(JSON_EXTRACT_SCALAR(event, "$.object.id"), 35) AS STRING) AS file_id,
                                         SUBSTR(JSON_EXTRACT_SCALAR(event, "$.membership.member.id"), 29) AS user_id,
                                         datetime(EVENT_TIME) as access_time
-                                        FROM learning_datasets.enriched_events
+                                        FROM event_store.events
```

Comment:
I thought this change was already made? Maybe this is just from an old commit?

Reply:
+1.